libraries/security/src/main/java/org/dataportabilityproject/security/RsaSymmetricKeyGenerator.java

```diff
@@ -26,7 +26,7 @@
 import org.slf4j.LoggerFactory;
 
 /** RSA-based implementation for {@link KeyPair} creation and encoding. */
-class RsaSymmetricKeyGenerator implements AsymmetricKeyGenerator {
+public class RsaSymmetricKeyGenerator implements AsymmetricKeyGenerator {
```

Comment:
TODO, provide a public module to bind KeyGenerators

Reply:
Done